applications/openshift/general/resource_requests_quota_per_project/oval/shared.xml

```diff
@@ -0,0 +1,94 @@
+  </unix:file_object>
+
+  <unix:file_test id="test_file_for_resource_requests_quotas_filtered_namespaces" check="all" check_existence="only_one_exists"
+    comment="Find the file to be checked ('{{{ openshift_filtered_path(namespaces_api_path, non_ctlplane_namespaces_filter) }}}')." version="1">
+    <unix:object object_ref="object_file_for_resource_requests_quotas_filtered_namespaces"/>
+  </unix:file_test>
+
+  <unix:file_object id="object_file_for_resource_requests_quotas_filtered_namespaces" version="1">
+    <unix:filepath var_ref="resource_requests_quotas_filtered_namespaces_file_location"/>
+  </unix:file_object>
+
+  <!-- Object gathering --> 
+  <ind:yamlfilecontent_object id="object_resource_requests_quota_per_project" version="1">
+    <ind:filepath var_ref="resource_requests_quota_per_project_file_location"/>
+    <ind:yamlpath>[:]</ind:yamlpath>
+  </ind:yamlfilecontent_object>
+  <local_variable comment="Items counter" datatype="int" id="local_variable_counter_resource_requests_quota_per_project" version="1">
+    <count>
+      <object_component object_ref="object_resource_requests_quota_per_project" item_field="value" record_field="#"/>
+    </count>
+  </local_variable>
+
+  <ind:yamlfilecontent_object id="object_resource_requests_quotas_filtered_namespaces" version="1">
+    <ind:filepath var_ref="resource_requests_quotas_filtered_namespaces_file_location"/>
+    <ind:yamlpath>[:].metadata.name</ind:yamlpath>
+  </ind:yamlfilecontent_object>
+  <local_variable comment="Items counter control" datatype="int" id="local_variable_counter_resource_requests_quotas_filtered_namespaces" version="1">
+    <count>
+      <object_component object_ref="object_resource_requests_quotas_filtered_namespaces" item_field="value" record_field="#"/>
+    </count>
+  </local_variable>
+  
+  <!-- Object counts -->
+  <ind:variable_test version="1" id="test_elements_count_for_resource_requests_quota_per_project" check="all"
+    comment="Count elements at both paths and compare">
+    <ind:object object_ref="object_elements_count_for_resource_requests_quota_per_project"/>
+    <ind:state state_ref="state_elements_count_for_resource_requests_quota_per_project"/>
+  </ind:variable_test>
+  
+  <ind:variable_object id="object_elements_count_for_resource_requests_quota_per_project" version="1">
+    <ind:var_ref>local_variable_counter_resource_requests_quota_per_project</ind:var_ref>
+  </ind:variable_object>
+
+  <!-- Comparison -->
+  <!-- This verifies that the filtered namespaces are equal to the namespaces with resourcequota -->
+  <ind:variable_state id="state_elements_count_for_resource_requests_quota_per_project" version="1">
+    <ind:value var_ref="local_variable_counter_resource_requests_quotas_filtered_namespaces"/>
+  </ind:variable_state>
+  
+  <!-- OCP data root declaration -->
+  <external_variable comment="Root of OCP data dump" datatype="string" id="ocp_data_root" version="1" />
+</def-group>
```

Comment:
Credit to @JAORMX, I found a nice way of doing checks from `applications/openshift/networking/configure_network_policies_namespaces`